packages/backend-api/src/api/assistant/index.ts

```diff
@@ -54,14 +53,7 @@ export function createAssistant(): express.Router {
       const scoreRequest = await CommentScoreRequest.findById(id);
 
       if (scoreRequest) {
-        const taskData = {
-          commentId: scoreRequest.get('commentId'),
-          userId: scoreRequest.get('userId'),
-          scoreData,
-        };
-
-        await enqueue<IProcessMachineScoreData>('processMachineScore', taskData, runImmediately || false);
-
+        await enqueueProcessMachineScoreTask(scoreRequest.get('commentId'), scoreRequest.get('userId'), scoreData, runImmediately);
```

Comment:
formatting for 80 character width maybe?

Reply:
We have no hope for hitting 80 characters:-( about half the files in the repository have lines longer than that.  I'm currently aiming at 120-130 characters which accommodate most files, and easily fits on a screen on a modern IDE.  Admittedly, the above line is borderline even with that criteria...